src/components/LineChart.jsx

```diff
@@ -3,22 +3,13 @@ import PropTypes from 'prop-types';
 import {Line} from 'react-chartjs-2';
 import moment from 'moment';
 
-const calculateHeight = (total) => {
-    if (total > 100000) return 275;
-    if (total > 50000) return 250;
-    if (total > 25000) return 225;
-    if (total > 10000) return 200;
-    if (total > 5000) return 175;
-    return 150;
-};
-
 const calculateMinMax = (data) => {
     const maxValue = Math.max(...data);
     // calculate a dynamic value to center the graph
     const scaleDifference = Math.pow(10, maxValue.toString().length-1);
     return {
         min: 0,
-        max: parseInt((maxValue/scaleDifference)+1)*scaleDifference // plus 1 to add more space in the top
+        max: Math.ceil((maxValue/scaleDifference) + 0.25)*scaleDifference // plus 0.25 to add more space in the top
```

Comment:
This can be done much more simply if we're getting rid of the Y axis labels, as the only thing this seems to accomplish having no weird (non-even, differently spaces) outlier as the max Y value.

Just use `maxValue*1.2` or so as the `max`?

Reply:
Fixed in the last commit.